pkgs/data/fonts/fira-mono/default.nix

```diff
@@ -3,17 +3,17 @@
 fetchzip {
   name = "fira-mono-3.206";
 
-  url = http://www.carrois.com/downloads/fira_mono_3_2/FiraMonoFonts3206.zip;
+  url = https://github.com/mozilla/Fira/archive/4.106.zip;
```

Comment:
This looks like a new version?

Reply:
fira and fira-mono are released together, but with different versions https://github.com/mozilla/Fira/releases/tag/4.106